modules/ocf_desktop/manifests/pulse.pp

```diff
@@ -19,4 +19,11 @@
     require => Package['libasound2-plugins'],
   }
 
+  $pulse_sink = lookup({'name' => 'pulse_sink', 'default_value' => nil})
```

Comment:
Ah, alright then, I'd try it without the default then, so just `lookup('pulse_sink')` and I think that should work fine.